build/make-olm.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+CSV_VERSION="${1:-0.0.0}"
+BUNDLE_DIR="manifests/node-maintenance-operator/"
+BUNDLE_DIR_VERSION="${BUNDLE_DIR}${CSV_VERSION}/"
```

Comment:
let's keep the tree structure same as the name listed in the CSV. Since the CSV is autogenerated, the only real option is changing the tree.

So it should look like

```
BUNDLE_DIR_VERSION="${BUNDLE_DIR}/v${CSV_VERSION}"
```

note the leading "v" -> it should be "vX.Y.Z" not "X.Y.Z"